sim/core/spell_effect.go

```diff
@@ -290,27 +290,22 @@ func (spellEffect *SpellEffect) applyAttackerModifiers(sim *Simulation, spell *S
 			spellEffect.BonusArmorPenRating += attacker.PseudoStats.BonusOHArmorPenRating
 		}
 	} else if spell.SpellSchool.Matches(SpellSchoolArcane) {
-		spellEffect.Damage *= attacker.PseudoStats.ArcaneDamageDealtMultiplier
+		spellEffect.Damage *= attacker.PseudoStats.ArcaneDamageDealtMultiplier * attacker.PseudoStats.MagicDamageDealtMultiplier
```

Comment:
This new pseudostat isn't necessary; just update all 5 schools. `MagicDamageDealtMultiplier` is only being used in 1 place, and it's during initialization where performance doesn't matter. This is adding another multiplication on the hot path.